MdeModulePkg/Universal/SetupBrowserDxe/Setup.c

```diff
@@ -3167,7 +3167,7 @@ ConfirmSaveFail (
 {
   CHAR16  *FormTitle;
   CHAR16  *StringBuffer;
-  UINT32  RetVal;
+  UINT32  RetVal = BROWSER_ACTION_UNREGISTER; // MU_CHANGE - Initialize variable that might not be updated due to error checking

```

Comment:
coding convention requires init independent of declaration.  I know we don't always follow this so you will see code that isn't compliant but for new changes we should. 

UINT32  RetVal;

RetVal = BROWSER_ACTION_UNREGISTER;

Reply:
Separated initialization and declaration.